### PR TITLE
UniversalAdId added for valid VAST 4.1

### PR DESCRIPTION
**Pull request by Vendin**
UniversalAdId added for valid VAST 4.1. UniversalAdId required in response beginning with VAST 4.0.

Paragraph 3.7.1
https://iabtechlab.com/wp-content/uploads/2018/11/VAST4.1-final-Nov-8-2018.pdf

### CODE REVIEW

**Comment by Vendin** [comment]
@sokil hey. I want to get feedback. I expect. Thank you.

File: src/Creative/AbstractLinearCreative.php [hunk 1]
```diff
@@ -231,7 +234,10 @@ public function setVideoClicksClickThrough($url)
 
         // update CData
         if ($clickThroughDomElement->hasChildNodes()) {
-            $clickThroughDomElement->replaceChild($cdata, $clickThroughDomElement->firstChild);
+            $clickThroughDomElement->replaceChild(
+                $cdata,
+                $clickThroughDomElement->getElementsByTagName('Linear')->item(0)
```
Comment (sokil) [inline src/Creative/AbstractLinearCreative.php]: Bug. Merge master and run tests

File: tests/DocumentTest.php [hunk 2]
```diff
@@ -27,6 +27,7 @@ public function testCreateInLineAdSection()
         $ad1
             ->createLinearCreative()
             ->setDuration(128)
+            ->setUniversalAdId('vendin.org', '15051996')
```
Comment (sokil) [inline tests/DocumentTest.php]: use something abstract like server.com or example.com

File: src/Creative/InLine/Linear.php [hunk 3]
```diff
@@ -106,7 +109,23 @@ public function skipAfter($time)
...
 
-        $this->getDomElement()->firstChild->setAttribute('skipoffset', $time);
+        $this->getDomElement()->getElementsByTagName('Linear')->item(0)->setAttribute('skipoffset', $time);
+
+        return $this;
+    }
+
+
+    /**
+     * @param int|string $idRegistry
```
Comment (sokil) [inline src/Creative/InLine/Linear.php:123]: Add version of vast from where this element is available and section of specification to read about it.

**Review by sokil — CHANGES_REQUESTED** [review]

File: src/Creative/AbstractLinearCreative.php (hunk 1 above)
Comment (Vendin) [reply src/Creative/AbstractLinearCreative.php]: Done

File: tests/DocumentTest.php (hunk 2 above)
Comment (Vendin) [reply tests/DocumentTest.php]: Done

File: src/Creative/InLine/Linear.php (hunk 3 above)
Comment (Vendin) [reply src/Creative/InLine/Linear.php:123]: Done

**Review by sokil — APPROVED** [review]

**Comment by Vendin** [comment]
Thank you for operativeness.

Update version ?

**Pull request merged**